### ISSUE TITLE
"Negative" white spaces in intersections in some letters

### ISSUE THREAD

**Issue by theblursed** [CatharsisFonts/Ysabeau#39]
Hello CatharsisFont,

I recently discovered your Ysabeau font on Google Fonts, and downloaded it. It is gorgeous. Compliments for your wonderful work. I appreciate not only the beauty of the design but also the richness of the glyphs set, which would make it perfect for linguistics, an important field for me.

Unfortunately, I have a problem, which in practice makes the font unusable for most texts. While the font shows correctly on Google Fonts, when downloaded in my computer (Mac) some "negative" white spaces appear in intersections of shapes in common letters, where they should be black (see picture). I have the same issue in Word, TextEdit, Font Book, LibreOffice.

I'm not an expert of fonts or coding (I actually just signed up here just to write this to you), but I like them a lot and have a lot, and I think this is the first time that I've found a difference between how the font appears on Google Fonts and how it appears on my computer.

Do you think it could be fixed?

Thank you and keep up the good work.

![Schermata 2023-05-25 alle 00 56 44](https://github.com/CatharsisFonts/Ysabeau/assets/134559730/4f595d4a-7c8c-446e-b2ef-9e471932756b)

**Comment by CatharsisFonts** [comment]
Not on my side...
<img width="152" alt="Screenshot 2023-05-25 at 09 12 20" src="https://github.com/CatharsisFonts/Ysabeau/assets/13754106/987a9e25-c233-4296-9d21-21c481b9bc33">

Which version are you using? OTF or TTF? Where did you download it?

Glad you like Ysabeau! :)

Cheers, Christian

**Comment by CatharsisFonts** [comment]
So at first sight, the fonts from Google Fonts work well on my machine, and I'm not getting any overlap issues. Do you get the same problems with the fonts from this repository?

**Comment by theblursed** [comment]
Yes, what I did was: Google Fonts > download whole family > unzip > "static" folder > install all fonts.

I now downloaded the files from the repository here, as you said.

If I install the .ttf files from the repository, the same issue appears as described, but _only_ for "plain" Ysabeau: Infant, Office and SC versions seem to work fine.

While, if I install the .otf files, _all_ fonts work fine, with no negative space (at least for what I've seen).

So, as a whole, the problem seems to lie only in the .ttf files (both on Google Fonts and on this repository) for "plain" Ysabeau, while everything else works fine.

Well, in practice this solves the problem for me... I could just use the .oft. Still, I'll give you any additional information you may need, if you want to address the problem somehow. 😃

**Comment by CatharsisFonts** [comment]
I suspect the reason why Vanilla Ysabeau works differently from the others is that you already had it installed — a font caching issue?

In any case, I'm not getting any undue effects from the GF Ysabeau. Still bears investigating; I've alerted the team responsible for the migration.

**Comment by theblursed** [comment]
I'm not an expert so I can't say, but before installing the files from the repository I uninstalled the ones from GF exactly to prevent this; and after, I also uninstalled the .ttf of the repository before installing the .oft ones.

**Comment by yanone** [comment]
This is a known rendering bug in older MacOS versions. It can only be circumvented by redrawing the outlines in a way that each glyph is composed of just one contour (on the outside, more contours for countershapes), with corners as shown in the image below.

There is a workaround to flag in the font to the MacOS renderer that a particular glyph needs special attention and then they will use another rasterizer for that glyph, but it will significantly slow down the rendering. Combined with fact that the issue has been solved in newer MacOS versions (forgot which one right now), and it can be solved in drawing, no one at Google Fonts has been paying much attention to this issue and honestly, I don't think anyone will. We can push an update if you care to redraw the outlines.

![Bildschirmfoto 2023-05-25 um 11 01 53](https://github.com/CatharsisFonts/Ysabeau/assets/175386/edc1da9c-8e31-467c-bdf7-ca12f58ca070)

**Comment by m4rc1e** [comment]
@theblursed what version of OS X are you using?

**Comment by theblursed** [comment]
@yanone I understand; I was already using the static version. However, if if the problem is only mine because of older software, no need to worry: as I said, I can use the .otf which seems to work fine. 🙂

@m4rc1e Catalina 10.15.7
